playbook.yml

```diff
@@ -1,9 +1,31 @@
 ---
 - hosts: all
+  gather_facts: False
 
   pre_tasks:
+
+    # Check Ubuntu release version to determine if we need to install python 2,
+    # an ansible dependency that isn't included by default in Ubuntu 16.04 and
+    # up.
+
+    - raw: cat /etc/lsb-release | grep DISTRIB_RELEASE | cut -d "=" -f2
+      register: ubuntu_release
+
+    - debug: msg="Running ubuntu version {{ ubuntu_release.stdout|float }}"
+
+  # Update apt cache and install python 2 for Ubuntu versions greater than 16.04
```

Comment:
Level of indentation for these comments appears to be a little less than the indentation for the comments above.